table/tables/tables_test.go

```diff
@@ -392,3 +392,94 @@ func (ts *testSuite) TestTableFromMeta(c *C) {
 	_, err = tb.AllocHandle(tk.Se)
 	c.Assert(err, NotNil)
 }
+
+func (ts *testSuite) TestHiddenColumn(c *C) {
+	tk := testkit.NewTestKit(c, ts.store)
+	tk.MustExec("CREATE DATABASE test_hidden;")
+	tk.MustExec("USE test_hidden;")
+	tk.MustExec("CREATE TABLE hidden (a int primary key, b int, c int, d int, e int);")
```

Comment:
ditto

Reply:
Since we create a new database, we don't need to check table hidden.